### PR TITLE
Updated Link by Stripe logo

### LINKED ISSUE #8366: Update Link by Stripe branding
### Description

On 8 March, we received the following communication from Stripe:

> The week of April 15th we plan to update the logos/icons used to the attached assets across:
> 
> - Logo and color swap on all Link product surfaces, including payment surfaces, Link app, Stripe dashboard, Stripe docs, support, Financial Connections, Identity, Crypto, and Stripe outbound emails
> - [link.com](http://link.com/) and [stripe.com/link](http://stripe.com/link) updates

### Acceptance criteria

Replacing the Link logo at `/assets/images/payment-methods/link.svg` (and any other Link-related branding) with updated assets.

### Designs
pbUcTB-4pX-p2

### Additional context

We have 7.5.0 launching during the week of 15th April, so it may be a good opportunity to update the branding in that release.

### PR DESCRIPTION
**Pull request by gpressutto5**
Fixes #8366

#### Changes proposed in this Pull Request

This PR updates the Link by Stripe logo according to the discussion on the original issue #8366.

image

image

#### Testing instructions

1. Enable Link by Stripe in WooPayments settings. While on it, ensure the logo is updated there.
2. Check Blocks Checkout and Shortcode Checkout and ensure the new logo is being displayed and it works as expected.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `8481` or branch name `update-link-by-stripe-logo` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=update-link-by-stripe-logo) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 912459508547291bb0c542628cb3c820cc39dcac
- Build time: 2024-04-04 17:48:24 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Review by mdmoore — APPROVED** [review]
LGTM!

**Comment by gpressutto5** [comment]
This PR is waiting for confirmation before being merged
p1711663461114469/1710878901.039989-slack-C9976E5MJ

**Pull request merged**
